src/sqlfluff/cli/commands.py

```diff
@@ -505,7 +512,8 @@ def lint(
         except OSError:
             click.echo(
                 colorize(
-                    f"The path(s) '{paths}' could not be accessed. Check it/they exist(s).",
+                    f"The path(s) '{paths}' could not be accessed. Check it/they "
+                    "exist(s).",
```

Comment:
You know the more I go through these sort of lines the more I think, black didn't care about these before, why are we changing these

Reply:
I guess black doesn't bother if it can't make a fix and usually I have flake8 set to complain about 88 line length so don't have to make all these changes at once. tbh this PR feels like changes for the sake of changes since the whole point of the config is we set and forget so who cares if the numbers don't match, it's still the same for everyone. I'll keep looking tho

Reply:
Yeah been there myself! Almost gave up on this.

The problem is `black` DOES care about code lines. And if `flake8` is set to 127, then it might spot them in our CI and so force contributor to run `black`. Until you add a line that's 128 chars or more and then you run `black` and get unrelated changes because other lines that are 100 chars long (so wasn't enough to flag in CI previously) are weirdly changed.

Additionally with the pre-commit we DO run `black` on every file at commit and, again, are getting unrelated changes flagged there for previously accepted lines.

Reply:
> I have flake8 set to complain about 88 line length

Others don't, as our default is 127. Hence the issue.

Reply:
No I'm using 127 like everyone else, I mean just in general I'm used to splitting my strings when they get to a certain length

Reply:
> The problem is `black` DOES care about code lines. And if `flake8` is set to 127, then it might spot them in our CI and so force contributor to run `black`

Everyone should just run black anyway and even if you don't run black it would raise the `BLK` flake8 lint. So if you have a code line that's 100 chars long and black could split it then flake8 would raise `BLK` and tell you to run black which would fix the line length. 

So the only thing reducing this flake8 line length will do is make us split docstrings and strings that black couldn't split

Reply:
> Everyone should just run black anyway

Yeah well can’t depend on that.

> and even if you don't run black it would raise the `BLK` flake8 lint. So if you have a code line that's 100 chars long and black could split it then flake8 would raise `BLK` and tell you to run black which would fix the line length.

Interesting! I wasn’t aware that’s what would happen. In that case I agree this PR is a lot less valuable and maybe not worth the effort.

> So the only thing reducing this flake8 line length will do is make us split docstrings and strings that black couldn't split.

Which, in an idea world, maybe isn’t a bad thing to enforce (as I said previously black chooses its line length for a reason - better readability). But being more pragmatic is perhaps more pain than it’s worth.

Hmm… I’m tempted to just close this then.

@chwiese what problems did you’d with this that led you to raise this?

Reply:
Before and after of how black and flake8 handles this:

Before:
![image](https://user-images.githubusercontent.com/80432516/150436847-be38e1df-8fac-4274-b51b-4b6b31cf8abe.png)
`BLK100` is raised, don't ask questions just run black.

After:
![image](https://user-images.githubusercontent.com/80432516/150436922-507a3f10-fef7-4378-b115-bb65e8fa5433.png)
After running black the only thing that's being flagged are really long strings/docstrings, which are fair enough to have to handle manually.

(to be clear I'm not against the flake8 line length being set to 88 early in a project, it's just seems like there's not much value in changing it now)

Reply:
Starting to change my mind about this again...

> (to be clear I'm not against the flake8 line length being set to 88 early in a project, it's just seems like there's not much value in changing it now)

Well the work is done now. So if our premise is we should have done it from beginning if we'd thought about it, but not worth the effort now, then the effort is (mostly) done - apart from code review.

Yes it does make some long strings a little more difficult, but then again some of them do flag (when > 127) anyway. And arguably shorter lines are easier for maintainability (e.g. git compares).

Going to have another look through these changes myself today but starting to veer towards merging again.

Reply:
@tunetheweb @barrywhart I'm fine with you guys merging given the work is done now 👍 

I was only pointing out that practically speaking it makes no difference and that I wasn't going to spend my thurs eve on it lol 😄

For interest: As @barrywhart mentioned 80 chars is more of a historical holdover: https://softwareengineering.stackexchange.com/questions/148677/why-is-80-characters-the-standard-limit-for-code-width

Reply:
I like Black's take on this: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

> You probably noticed the peculiar default line length. Black defaults to 88 characters per line, which happens to be 10% over 80. This number was found to produce significantly shorter files than sticking with 80 (the most popular), or even 79 (used by the standard library). In general, 90-ish seems like the wise choice.
>
> ...
> 
> You can also increase it, but remember that people with sight disabilities find it harder to work with line lengths exceeding 100 characters. It also adversely affects side-by-side diff review on typical screen resolutions. Long lines also make it harder to present code neatly in documentation or talk slides.

I was always against line length limits previous (we have bigger screens! and line wrap in our IDs!) but that's won we round a bit.

I also tried to increase it, but could only get it to work with command line option, rather than with a `pyproject.toml` so gave up.

Anyway, now we've sorted the Doc issue, and I've also just fixed some URL issues (I didn't like breaking them - but looks like Black/Flake8 doesn't complain if it's the only thing on a line so can't be broken) I think good to merge and try to encourage shorter line lengths (for above reasons).

Anyway - thanks for letting me know you're OK @jpy-git . Happy to know I won't be totally going against your wishes my merging 😁